### PR TITLE
Fixes for AZ review page

### PR DESCRIPTION
**Pull request by rickreyhsig**
https://www.pivotaltracker.com/story/show/186892172

### Ask whether incarcerated
Screen Shot 2024-03-07 at 12 26 42 PM

### CODE REVIEW

File: app/models/state_file_base_intake.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ class StateFileBaseIntake < ApplicationRecord
 
   scope :accessible_intakes, -> { all }
   devise :timeoutable, :timeout_in => 15.minutes, :unlock_strategy => :time
+  # devise :timeoutable, :timeout_in => 15.days, :unlock_strategy => :time
```
Comment (rickreyhsig) [inline app/models/state_file_base_intake.rb:24]:
```suggestion
```

File: config/locales/en.yml [hunk 2]
```diff
@@ -2021,7 +2021,6 @@ en:
           charitable_contributions: Made charitable contributions
           charitable_noncash: Charitable contributions (Non-Cash)
           dependent_tax_credit: Dependent tax credit
-          excise_credit: Incarcerated in a county, state, or federal prison for at least 60 days in 2023
```
Comment (tofarr) [inline config/locales/en.yml:2024]: I did a search and did not find any other usages of this key

File: app/views/state_file/questions/az_review/edit.html.erb [hunk 3]
```diff
@@ -45,8 +45,14 @@
   <% if current_intake.ask_whether_incarcerated? %>
     <div id="incarcerated" class="blue-group">
       <div class="spacing-below-5">
-        <p class="text--bold spacing-below-5"><%=t(".excise_credit") %></p>
+        <p class="text--bold spacing-below-5"><%=t(".was_incarcerated") %></p>
```
Comment (tofarr) [inline app/views/state_file/questions/az_review/edit.html.erb:48]: I did not find a key `state_file.questions.az_review.edit.was_incarcerated` - Did I miss it?

**Review by tofarr — CHANGES_REQUESTED** [review]

File: config/locales/en.yml (hunk 2 above)
Comment (rickreyhsig) [reply config/locales/en.yml:2024]: https://github.com/codeforamerica/vita-min/pull/4303

**Pull request merged**